java/ComputerVision/ComputerVisionQuickstart.java

```
@@ -7,69 +8,66 @@
-     *  - Displaying any landmarks detected in the image and their bounding boxes
-     *  - Displaying what type of clip art or line drawing the image is
-     *
-     */
-
-
-    public static void RunQuickstarts()
     {
         /*  Configure the local environment:
          *
-         *  Set the AZURE_COMPUTERVISION_API_KEY and AZURE_REGION environment variables on your
+         *  Set the AZURE_COMPUTERVISION_API_KEY environment variable on your
          *  local machine using the appropriate method for your preferred command shell.
          *
-         *  For AZURE_REGION, use the same region you used to get your subscription keys.
-         ***Can we link (or just put the URL here) to the docs for finding your region?
-         *
          *  Note that environment variables cannot contain quotation marks, so the quotation marks
          *  are included in the code below to stringify them.
          *
          *  Note that after setting these environment variables in your preferred command shell,
          *  you will need to close and then re-open your command shell.
          */
-
         String azureComputerVisionApiKey = System.getenv("AZURE_COMPUTERVISION_API_KEY");
-        String azureRegion = System.getenv("AZURE_REGION");
         //  END - Configure the local environment.
 
 
         /*  Create an authenticated Computer Vision client:
          *
-         *  Concatenate the Azure region with the Azure base URL to create the endpoint URL, and
-         *  then create an authenticated client with the API key and the endpoint URL.
+         *  Create the endpoint URL. You may have to change the first part ("westus") 
+         *  to match your subscription
+         *  Then create an authenticated client with the API key and the endpoint URL.
          */
-
-        String endpointUrl = ("https://").concat(azureRegion).concat(".api.cognitive.microsoft.com");
+        String endpointUrl = ("https://westus.api.cognitive.microsoft.com");
```

Hey Patrick, Feedback from our user studies suggest we should not pre-fill the endpoint because it led to silent failures where people didn't notice they needed to set the region / endpoint based. This was one of the most common issues we observed.

---

Fixed